app/controllers/api/tables_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::TablesController < ApplicationController
+  def create
+    name = params.require(:name)
+    if name.blank?
+      render json: { errors: 'The table name is required.' }, status: :unprocessable_entity
+    elsif name.length > 50
+      render json: { errors: 'You cannot input more 50 characters.' }, status: :unprocessable_entity
+    else
```

Comment:
The `Api::TablesController` is using a manual way to validate the presence and length of the `name` parameter. Consider using ActiveModel validations to make the code cleaner and more maintainable.